app/controllers/storage_controller.rb

```diff
@@ -462,10 +462,6 @@ def breadcrumb_name(_model)
     _("Datastores")
   end
 
-  def tagging_explorer_controller?
```

Comment:
@mzazrivec if you remove this method, you will not be able to load Tagging form for a selected Datastore.